content/en/users/cloud-compute/faq/_index.md

```diff
@@ -162,42 +180,39 @@ users:
       - <paste here the contents of your SSH key pub file>
 ```
 
-{{% alert title="Warning" color="warning" %}}
-YAML format requires that the spaces at the beginning of each line is
-respected in order to be correctly parsed by `cloud-init`.
-{{% /alert %}}
+{{% alert title="Warning" color="warning" %}} YAML format requires that the
+spaces at the beginning of each line is respected in order to be correctly
+parsed by `cloud-init`. {{% /alert %}}
 
 ### How can I use a contextualisation file?
 
-If you have a contextualisation file, you can use it with the
-`--user-data` option to `server create` in OpenStack.
+If you have a contextualisation file, you can use it with the `--user-data`
+option to `server create` in OpenStack.
 
-``` {.shell}
-$ openstack server create --flavor <your-flavor> --image <your image> \
-            --user-data <your contextualisation file> \
-            <server name>
+```shell
+openstack server create --flavor <your-flavor> --image <your image> \
+          --user-data <your contextualisation file> \
+          <server name>
 ```
 
-{{% alert title="Note" color="info" %}}
-We recommend using `cloud-init` for contextualisation. EGI images in
-AppDB do support `cloud-init`. Check the
-[documentation](https://cloudinit.readthedocs.io/) for more
-information.
+{{% alert title="Note" color="info" %}} We recommend using `cloud-init` for
+contextualisation. EGI images in AppDB do support `cloud-init`. Check the
+[documentation](https://cloudinit.readthedocs.io/) for more information.
 {{% /alert %}}
 
 ### How can I pass secrets to my VMs?
 
-EGI Cloud endpoints use https so information passed to contextualise the
-VMs can be assumed to be safe and only readable within your VM. However,
-take into account that anyone with access to the VM may be able to
-access also the contextualisation information.
+EGI Cloud endpoints use https so information passed to contextualise the VMs can
+be assumed to be safe and only readable within your VM. However, take into
+account that anyone with access to the VM may be able to access also the
```

Comment:
Maybe it would be good to add an info/warning box for this security-related point

Reply:
Any suggestion on what to include there besides what's already written?

Reply:
It could be enough, or if you want to expand you could provide an example such as:

> Take into account that anyone with access to the VM may be able to access also the contextualisation information, so ensure that no sensitive data like clear test passwords is used during contextualisation.

Reply:
Done